### DISCUSSION TITLE
Help more people discover dicom-rs.

### DISCUSSION THREAD

**Discussion by bowenxuuu** [Enet4/dicom-rs#554] [General]
Hello Enet4,

Thank you for your contributions to `dicom-rs`. I intend to share `dicom-rs` in the [rust-unofficial/awesome-rust](https://github.com/rust-unofficial/awesome-rust) repository, which is a curated list of Rust code and resources. This would help more people discover `dicom-rs`. What are your thoughts on this?

**Comment by Enet4** [comment]
Hello @xb284524239! Thank you for your interest in the project! Proposing to add this repository to lists of that kind is pretty much fine by me. I happen to know that DICOM-rs was added to other similar lists already, such as [awesome-dicom](https://github.com/open-dicom/awesome-dicom?tab=readme-ov-file#rust), by folks external to the project.

**Reply by bowenxuuu** [reply]
Okay, I've submitted a pull request to the [rust-unofficial/awesome-rust](https://github.com/rust-unofficial/awesome-rust) repository. Now we just need to wait for it to be reviewed and merged.
